### PR TITLE
sample config with doc

### PR DESCRIPTION
**Pull request by davidkel**
This is a sample core.yaml file for fsc and fabric-driver, for discussion to add and improve the associated documentation
Signed-off-by: Dave Kelsey <d_kelsey@uk.ibm.com>

### CODE REVIEW

File: exampleconfigs/core-fabric-example.yaml [hunk 1]
```diff
@@ -0,0 +1,328 @@
...
+    # and that entry must have an address with an entry P2P.
+    bootstrapNode: approver
+
+  # ------------------- KVS Configuration -------------------------
+  # Internal key/value store used by the node to store information
+  # such as bindings (eg resolvers)
+  kvs:
+    cache:
+      # Sets the maximum number of cached items
+      size:
```
Comment (mbrandenburger) [inline exampleconfigs/core-fabric-example.yaml:87]: empty size, will it be set to 0 in that case?

**Review by mbrandenburger — COMMENTED** [review]
Great step forward! The example config has a few references to the `approver` identity. If one does not carefully read, it might be understood as a default value or a FSC specific role.
Should we replace them with some other generic identities? E.g., `someOtherId`? Alternative we should also briefly explain the setting where we have a node that uses this config and another one having `fsc.id: approver`. WDYT?

File: exampleconfigs/core-fabric-example.yaml (hunk 1 above)
Comment (davidkel) [reply exampleconfigs/core-fabric-example.yaml:87]: That's a good point, if any of the entries doesn't have a value, what is it's default. I will look at adding those

File: exampleconfigs/README.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# Core.yaml examples
```
Comment (adecaro) [inline exampleconfigs/README.md:1]: shall we link this document from the documentation we have under `./doc`?

File: exampleconfigs/core-fabric-example.yaml [hunk 3]
```diff
@@ -0,0 +1,343 @@
+---
```
Comment (adecaro) [inline exampleconfigs/core-fabric-example.yaml:1]: we need to add a section also for Orion

**Comment by davidkel** [comment]
Closing as #427 supercedes this

**Pull request closed without merging**